gradlew.bat

```diff
@@ -43,11 +43,11 @@ set JAVA_EXE=java.exe
 %JAVA_EXE% -version >NUL 2>&1
 if %ERRORLEVEL% equ 0 goto execute
 
-echo.
-echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
-echo.
-echo Please set the JAVA_HOME variable in your environment to match the
-echo location of your Java installation.
+echo. 1>&2

```

Comment:
Any idea why do we need this change.

Reply:
That is automatically updated when we run the `./gradlew wrapper` to upgrade the wrapper version. We need to upgrade the wrapper to upgrade AGP

Reply:
Got it. Thanks for the info!